.eslintrc

```diff
@@ -16,18 +16,20 @@
   "overrides": [
     {
       // We exclude test files from dependency injection format checks because
-      // they are never minified.
-      "files": ["*Spec.js", "*Spec.ts"],
+      // they are never minified. Also, we exclude no-loss-of-precision check as
+      // it is only relevant for non-test files.
```

Comment:
Why it's only relevant to non-test files? Are we expecting to allow tests to use numbers that lose precision on runtime? (In general, should we expect consistency in the structure of numbers in actual-code and test-code?)

Reply:
Updated the comment.